scripts/mkcert.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+
+# Install mkcert's root certificate
+mkcert -install &&
+
+	# Add mkcert's root certificate into the local python certificate store
+	cat $(mkcert -CAROOT)/rootCA.pem >>$(python -m certifi)
+
+# Geneate local certificates for dev
+mkcert -key-file config/localhost-key.pem -cert-file config/localhost.pem localhost 127.0.0.1 dev.getsentry.net *.dev.getsentry.net
```

Comment:
not sure we still need localhost?
```suggestion
mkcert -key-file config/localhost-key.pem -cert-file config/localhost.pem dev.getsentry.net *.dev.getsentry.net
```

Reply:
We probably don't anymore. Might be a good idea to completely remove it.